libvast/vast/view.hpp

```diff
@@ -239,11 +239,6 @@ class container_view_handle
 
   template <class Hasher>
   friend void hash_append(Hasher& h, container_view_handle xs) {
-    if (!xs) {
-      hash_append(h, false);
-      return;
-    }
-    hash_append(h, true);
```

Comment:
Maybe add an assertion that `xs` is not null? Otherwise `*xs` below may crash.

Reply:
Instead I added the hash of `caf::none` that was there originally; after some consideration I felt that it would be weird if the hash function can crash for a valid container_view state, even if we should not hash a container_view in that state.